fireapp/lib/presentation/volunteer_information/volunteer_information.dart

```diff
@@ -0,0 +1,512 @@
+                    Text(
+                      'Volunteer Preference Hour:',
+                      style: Theme
+                          .of(context)
+                          .textTheme
+                          .bodyMedium,
+                      textAlign: TextAlign.left,
+                    ),
+                    Expanded(
+                      child: Text(
+                        AppLocalizations.of(context)?.volunteer_prefHours(
+                            volunteerInformation.prefHours) ??
+                            'Volunteer Address',
+                        style: Theme
+                            .of(context)
+                            .textTheme
+                            .bodyMedium,
+                        textAlign: TextAlign.right,
+                      ),
+                    ),
+                  ],
+                ),
+                Padding(
+                    padding: EdgeInsets.only(bottom: 16.0),
+                    child: Row(
+                      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                      children: [
+                        Text(
+                          'Volunteer Experience:',
+                          style: Theme
+                              .of(context)
+                              .textTheme
+                              .bodyMedium,
+                          textAlign: TextAlign.left,
+                        ),
+                        Expanded(
+                          child: Text(
+                            AppLocalizations.of(context)?.volunteer_expYears(
+                                volunteerInformation.expYears) ??
+                                'Volunteer Experience',
+                            style: Theme
+                                .of(context)
+                                .textTheme
+                                .bodyMedium,
+                            textAlign: TextAlign.right,
+                          ),
+                        ),
+                      ],
+                    ),
+
+                ),
+                Padding(
+                  padding: EdgeInsets.only(bottom: 16.0),
+                  child: Row(
+                    mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                    children: [
+                      Text(
+                        'Volunteer Possible Roles:',
+                        style: Theme
+                            .of(context)
+                            .textTheme
+                            .bodyMedium,
+                        textAlign: TextAlign.left,
+                      ),
+                      Expanded(
+                        child: Text(
+                          (AppLocalizations.of(context)?.volunteer_possibleRoles
+                            (volunteerInformation.possibleRoles)
+                              ?? 'Volunteer Possible Roles')
+                              .toString()
+                              .replaceAll('[', '')
```

Comment:
Do _not_ call `toString`. Do not replace the array syntax (`[`, `]`, `,`). If you're doing either of those in user facing text, it means you're doing something wrong. 

Put each element of the list in a `Text` widget inside a column.